sr_gui_change_controllers/src/sr_gui_change_controllers/change_controllers.py

```diff
@@ -135,7 +156,15 @@ def confirm_current_control(self):
             rospy.logerr(error + ". Error: " + err)
             return
 
-        running_controllers = [c for c in resp1.controller if c.state == "running"]
+        # Confirm all running controllers for the controls in this GUI
+        running_controllers = [c for c in resp1.controller
```

Comment:
Could we use a better name than `c`?

Reply:
actually, and `resp1`

Reply:
fixed in [542d322](https://github.com/shadow-robot/sr_visualization_common/pull/35/commits/542d322c257524753a6d7f478c22fe42ffc70d69)